src/catalog/catalog_content.cpp

```diff
@@ -181,7 +181,7 @@ void CatalogContent::saveToFile(const std::string& directory, FileVersionType db
         std::make_unique<BufferedFileWriter>(vfs->openFile(catalogPath, O_WRONLY | O_CREAT)));
     writeMagicBytes(serializer);
     serializer.serializeValue(StorageVersionInfo::getStorageVersion());
-    serializer.serializeValue(tableSchemas.size());
+    serializer.serializeValue((uint64_t)tableSchemas.size());
```

Comment:
Should we use uint32_t here? should be enough

Reply:
Since the current version uses `uint64_t`, I prefer to just stick to it instead of creating a new storage version for this.